src/Resources/views/Invoice/Download/pdf.html.twig

```diff
@@ -42,19 +40,19 @@
                         {{ item.variantCode }}
                     </td>
                     <td>
-                        {{ money.format(item.unitPrice, invoice.currencyCode) }}
+                        {{ item.unitPrice|sylius_format_money(invoice.currencyCode, sylius.localeCode) }}
```

Comment:
Could we also save `localeCode` on an invoice? We can take it directly from an order instance.

Reply:
I think that we could. However, it would create an issue and do it in a separate PR.

Reply:
#13